extensions/blocks/podcast-player/components/podcast-player.js

```diff
@@ -246,7 +264,12 @@ export class PodcastPlayer extends Component {
 					id={ `jetpack-podcast-player__tracklist-title--${ playerId }` }
 					className="jetpack-podcast-player--visually-hidden"
 				>
-					{ /* translators: %s is the track title. This describes what the playlist goes with, like "Playlist: [name of the podcast]" */ }
+					{ /*
+					 * This describes what the playlist goes with, like "Playlist: [name
+					 * of the podcast]".
+					 *
+					 * translators: %s is the track title.
+					 */ }
```

Comment:
Have you tested if this comment annotation gets picked up as expected?

Reply:
Fixed in https://github.com/Automattic/jetpack/pull/15281/commits/116bc2b09cbb167da38709b588dc5e4296ed4b5f